torchbenchmark/models/lennard_jones/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+devices:
+  NVIDIA A100-SXM4-40GB:
+    eval_batch_size: 1000
```

Comment:
@xuzhao9 I'm not sure how this is supposed to be determined, I just put the default batch size that the above script runs with.

Reply:
Yeah the default batch size is fine. We have a script to generate the optimal batch size for different devices based on FLOPS result, but we can save that work later.